### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Recompare pop up_Keyboard: After activating the popup "Recompare" keyboard focus order is not logical

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6661]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #MAS2.4.3; #Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAS reference: MAS2.4.3

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Options menu from Schema compare Menu bar & hit enter.
12. Check or uncheck some options & hit Ok.
13. Validate Re-compare pop will open
14. Use tab key to navigate to the control of the popup and check the navigation

**Actual:**
After activating the popup "Re-compare" keyboard focus is moving to "Options button" , it does not move inside the popup.

**Same issue is observed in the following page:**
1.SchemaCompareOptions_SchemaCompare
Repeat 1-11 steps and press tab key
When clicked on 'Options' button, Schema compare options dialog opens but if pressed tab key, focus doesn't move to 1st control of the dialog but to background controls first and then comes to opened dialog. 
Here user expectation is when clicked on 'Options' button, focus should be moved to 1st control of Schema compare options dialog.

**Expected:**
After activating the popup keyboard focus should move "Inside the Re-compare Popup".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Keyboard user will not be able to know the flow of the screen.

**MAS Reference:**
MAS2.4.3:[ https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}]( https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9})

**Attachment for Reference:**
![Accessibility A11y_AzureDataStudio(Schema,Dacpac,Backup)_Recompare pop up_Keyboard](https://user-images.githubusercontent.com/24580570/62776977-ae7aff80-bac9-11e9-865c-f88a36d8cd76.png)

**Comment by udeeshagautam** [comment]
Same behavior as VScode

**Comment by hannahqin** [comment]
Reopening and reassigning to @fsteffi as per the [Accessibility External Bug Process](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). @fsteffi, this is an upstream issue from VS Code. Per the instructions in the External Bug Process, can you file this bug with vscode and track it there? https://github.com/microsoft/vscode/issues

**Comment by hannahqin** [comment]
Marking this for Exclusion. @fsteffi, please work with a SME to review.
https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx

**Importance of Experience:** minimal
Schema compare feature is an extension, not part of ADS core, thus does not affect all users. The re-compare pop-up provides a convenient reminder and easy access point for re-comparing after modifying options, but not necessary to the scenario. Users could easily click "Compare" to achieve the same end result.

**Quality of Experience:** not impacted
After modifying options, instead of using the controls in the pop-up, users can directly select "Compare" to recompare.

**AAT Conformance Doc:** https://cela-ra-aat-prod.azurewebsites.net/#/workflow/ee5c9007-47f4-4fcb-a4d7-a998a617edde?tab=default

**Comment by fsteffi** [comment]
@hannahqin we have logged external : https://github.com/microsoft/vscode/issues/81483

**Comment by hannahqin** [comment]
This was re-filed with vscode as microsoft/vscode#81786

**Comment by kenvanhyning** [comment]
Closing as upstream

**Comment by Alexandria-R** [comment]
Not approved, as this is an improper use of toast and was not acknowledged by the external party.

**Comment by hannahqin** [comment]
@Alexandria-R The VSCode (external) team is saying that they do not think toast notifications should steal focus, and that we should use a large modal dialog if we want focus to shift immediately. 

We are using toast notifications as expected according to VSCode's documentation and believe that it's the best user experience to use a toast notification in these cases instead of a modal dialog. However, we don't have control over where the keyboard focus moves when the toast notification pops up, as described in this issue filed by @fsteffi. 

Assuming that we are using toast notifications as VSCode intends, which we are, and you believe that the current behavior of keyboard focus is an accessibility issue, then this needs to be taken up with the VSCode team. We unfortunately do not have control over this.
